scripts/core/menu/menu.ts

```diff
@@ -1,5 +1,6 @@
 import {reactToAngular1} from 'superdesk-ui-framework';
 import {GlobalMenuHorizontal} from './GlobalMenuHorizontal';
+import { flags } from 'apps/search/components/fields/flags';
```

Comment:
Is this used anywhere?

Reply:
It's not, added by VS Code automatically... Removing...